release-notes/download-archives/2.0.7-download.md

```diff
@@ -0,0 +1,170 @@
+| macOS   | [64-bit](https://download.microsoft.com/download/2/E/C/2EC018A0-A0FC-40A2-849D-AA692F68349E/dotnet-sdk-2.1.105-osx-x64.pkg)  | [64-bit](https://download.microsoft.com/download/2/E/C/2EC018A0-A0FC-40A2-849D-AA692F68349E/dotnet-sdk-2.1.105-osx-x64.tar.gz)| [64-bit](https://download.microsoft.com/download/A/9/F/A9F8872C-48B2-41DB-8AAD-D5908D988592/dotnet-runtime-2.0.7-osx-x64.pkg)      | [64-bit](https://download.microsoft.com/download/A/9/F/A9F8872C-48B2-41DB-8AAD-D5908D988592/dotnet-runtime-2.0.7-osx-x64.tar.gz)   |
+| Linux * | See installations steps below                         | [64-bit](https://download.microsoft.com/download/2/E/C/2EC018A0-A0FC-40A2-849D-AA692F68349E/dotnet-sdk-2.1.105-linux-x64.tar.gz)     | -                                                                  | [64-bit](https://download.microsoft.com/download/A/9/F/A9F8872C-48B2-41DB-8AAD-D5908D988592/dotnet-runtime-2.0.7-linux-x64.tar.gz) |
+
+**Checksum** files to verify downloads are available as follows:
+* [Checksums_Runtime](https://dotnetcli.blob.core.windows.net/dotnet/checksums/2.0.7-runtime-sha.txt)
+* [Checksums_SDK](https://dotnetcli.blob.core.windows.net/dotnet/checksums/2.1.105-SDK-sha.txt)
+
+**Debug Symbols**
+* [Shared Framework](https://download.microsoft.com/download/A/9/F/A9F8872C-48B2-41DB-8AAD-D5908D988592/corefx-2.0.7-symbols.zip)
+* [Runtime](https://download.microsoft.com/download/A/9/F/A9F8872C-48B2-41DB-8AAD-D5908D988592/coreclr-2.0.7-symbols.zip)
+
+## Docker
+
+Images for .NET Core 2.0.7 are available on [Docker](https://hub.docker.com/r/microsoft/dotnet/).
+## Installing .NET Core on Linux
+
+### Uninstall any Preview versions
+
+If you have .NET Core Previews installed, it will need to be removed before attempting to install .NET Core 2.0. This can be done by running a command such as the following.
+
+`sudo apt remove dotnet-dev-2.0.0-preview2-006497`
+
+### Ubuntu and Debian based systems
+
+Register the Microsoft key, the product repository for your distro and install required system dependencies with the following scripts.
+
+#### Ubuntu 18.04
+
```

Comment:
First, we don't have 18.04 signed packages yet so should maybe exclude this section for now. 

Second, there is no config .deb for 18.04 yet. Steps need to be the manual variety: 

+```bash
+wget -qO- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.asc.gpg
+sudo mv microsoft.asc.gpg /etc/apt/trusted.gpg.d/
+wget -q https://packages.microsoft.com/config/ubuntu/18.04/prod.list
+sudo mv prod.list /etc/apt/sources.list.d/microsoft-prod.list
+```

Reply:
Updated